e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/user-settings-persistence.cy.spec.ts

```diff
@@ -0,0 +1,213 @@
+import { enableJwtAuth } from "e2e/support/helpers/e2e-jwt-helpers";
+
+import { getEmbedSidebar, navigateToEntitySelectionStep } from "./helpers";
+
+const { H } = cy;
+
+describe("scenarios > embedding > sdk iframe embed setup > user settings persistence", () => {
+  beforeEach(() => {
+    H.restore();
+    cy.signInAsAdmin();
+    H.activateToken("bleeding-edge");
+
+    cy.intercept("GET", "/api/dashboard/**").as("dashboard");
+    cy.intercept("POST", "/api/card/*/query").as("cardQuery");
+  });
+
+  it("persists dashboard embed options", () => {
+    navigateToEmbedOptionsStep({ experience: "dashboard" });
+
+    cy.log("1. set embed settings to non-default values");
+    getEmbedSidebar().within(() => {
+      cy.findByLabelText("Allow downloads").click().should("be.checked");
+
+      capturePersistSettings();
```

Comment:
We want to intercept the second request, so we can be sure that both settings are persisted. The `waitAndReload()` helper function waits for the `PUT /api/setting/sdk-iframe-embed-setup-settings` to finish first.